BootloaderCorePkg/Tools/SblFwUpdate.py

```diff
@@ -0,0 +1,42 @@
+## @ SblFwUpdate.py
+#
+# Copyright (c) 2017-2020, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+import os
+import re
+import sys
+import shutil
+import subprocess
+
+def Usage():
+    print ("SblFwUpdate")
+    print ("Usage:")
+    print ("    SblFwUpdate  <Input Capsule File>")
+
+def Main():
+    #
+    # Parse the options and args
+    #
+    argc = len(sys.argv)
+    if argc != 2:
+        Usage()
+        return 1
+
+    if not os.path.exists(sys.argv[1]) :
+        raise Exception ('%s file not found !' % sys.argv[1])
+
+    if not os.path.exists('/boot/efi/'):
+        raise Exception ('/boot/efi/ path not found !')
+
+    shutil.copy(sys.argv[1], '/boot/efi/FwuImage.bin')
+    os.system('echo 1 > /sys/bus/wmi/devices/44FADEB1-B204-40F2-8581-394BBDC1B651/firmware_update_request')
+    output = subprocess.check_output(['cat','/sys/bus/wmi/devices/44FADEB1-B204-40F2-8581-394BBDC1B651/firmware_update_request']).decode()

```

Comment:
Let's use subprocess instead of os.system which is blocking call.

```
fwu_sysfs = '/sys/bus/wmi/devices/44FADEB1-B204-40F2-8581-394BBDC1B651/firmware_update_request'
subprocess.call(['echo', '1', '>', fwu_sysfs]);
subprocess.check_output(['cat', fwu_sysfs]).decode()
```

Reply:
I could not use call but I used subprocess.Popen

Reply:
I could not use call but I used subprocess.Popen, fixed this in new patch